bin/reference_data.py

```diff
@@ -325,6 +224,194 @@ def now():
         return datetime.utcnow().strftime('%d_%m_%Y_%H:%M:%S')
 
 
+class EnsemblDownloader(Downloader):
+    ftp_site = 'ftp.ensembl.org'
+    rest_site = 'http://rest.ensembl.org'
```

Comment:
The values of the variable here are very much attached to the class and is very unlikely to change independently of the rest of the class. It also provide clarity about where we getting the data from.

Reply:
Then perhaps we could assign the value to a constant and then use the constant in the code?

Reply:
There is no constant in python that I know of. If you mean that the variable could be in a separate file called `contants.py` then it would decrease code readability and increase complexity for no gain.

Reply:
I see what you mean. In past experience, literals like this URL string would have been stored in a variable such as FTP_SITE_URL at the top of the file, and then that variable would be used in the code itself. It's not a constant in the classical sense, although capitalised variable names have come to mean that in Python.